en/discover/deployment/articles-dxp/02-installing-search-engine/02-enterprise-search/01-security/00-security-intro.markdown

```diff
@@ -171,3 +287,61 @@ Here's the complete list of security configuration options:
 
 When you're finished configuring X-Pack Security, restart Elasticsearch. These
 steps require a full cluster restart.
+
+## Example Elasticsearch Security Configurations
+
+Here are the complete Elasticsearch 7.x configurations from this article (`elasticsearch.yml`; applies equally to Elasticsearch 6.5.x+):
+
+### Example Elasticsearch Security Configuration - PKCS#12
+
+```yaml
+cluster.name: LiferayElasticsearchCluster
+
+# X-Pack Security
+xpack.security.enabled: true
+
+## TLS/SSL settings for Transport layer (PKCS#12)
+xpack.security.transport.ssl.enabled: true
+xpack.security.transport.ssl.keystore.path: certs/elastic-nodes.p12
+xpack.security.transport.ssl.keystore.password: liferay
+xpack.security.transport.ssl.truststore.path: certs/elastic-nodes.p12
+xpack.security.transport.ssl.truststore.password: liferay
+xpack.security.transport.ssl.verification_mode: certificate
+
+# TLS/SSL settings for HTTP layer (PKCS#12)
+xpack.security.http.ssl.enabled: true
+xpack.security.http.ssl.keystore.path: certs/elastic-nodes.p12
+xpack.security.http.ssl.keystore.password: liferay
+xpack.security.http.ssl.truststore.path: certs/elastic-nodes.p12
+xpack.security.http.ssl.truststore.password: liferay
+
+# Comment out when Kibana and Liferay's LES Monitoring are also configured
```

Comment:
Do you mean to say _uncomment_ (remove the leading `#`) or did you really mean _Comment out_ (add the leading `#`)?

Reply:
I propose to change it to 

```properties
# Use this setting when Kibana and Liferay's LES Monitoring are also configured
#xpack.monitoring.collection.enabled: true
```